tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1683,6 +1771,93 @@ def test_copy_pds_member_with_system_symbol(ansible_zos_module,):
         hosts.all.zos_data_set(name=dest, state="absent")
 
 
+@pytest.mark.pdse
+def test_copy_pds_loadlib_member_to_pds_loadlib_member(ansible_zos_module,):
+    hosts = ansible_zos_module
+    # The volume for this dataset should use a system symbol.
+    # This dataset and member should be available on any z/OS system.
+    src = "USER.LOAD.SRC"
+    dest = "USER.LOAD.DEST"
+    cobol_pds = "USER.COBOL.SRC"
+    try:
+        hosts.all.zos_data_set(
+            name=src,
+            state="present",
+            type="pdse",
+            record_format="U",
+            record_length=0,
+            block_size=32760,
+            space_primary=40,
+            space_type="M",
+            replace=True
+        )
+
+        hosts.all.zos_data_set(
+            name=dest,
+            state="present",
+            type="pdse",
+            record_format="U",
+            record_length=0,
+            block_size=32760,
+            space_primary=40,
+            space_type="M",
+            replace=True
+        )
+
+        hosts.all.zos_data_set(
+            name=cobol_pds,
+            state="present",
+            type="pds",
+            space_primary=40,
+            record_format="FB",
+            record_length=80,
+            block_size=3120,
+            replace=True,
+        )
+        member = "HELLOSRC"
+        cobol_pds = "{0}({1})".format(cobol_pds, member)
+        rc = hosts.all.zos_copy(
+            content=COBOL_SRC,
+            dest=cobol_pds,
+        )
+        dest_name = "{0}({1})".format(dest, member)
+        src_name = "{0}({1})".format(src, member)
+        
+        
+        # both src and dest need to be a loadlib
+        rc = link_loadlib_from_cobol(hosts, dest_name, cobol_pds)
+        assert rc == 0
+        rc = link_loadlib_from_cobol(hosts, src_name, cobol_pds)
+        assert rc == 0
+
+        copy_res = hosts.all.zos_copy(
+            src="{0}({1})".format(src, member), 
+            dest="{0}({1})".format(dest, "MEM1"), 
+            remote_src=True)
+
+        verify_copy = hosts.all.shell(
```

Comment:
While this is a good check, one thing I think this test case is missing is did the load module copy successfully as in it still executable after its copied. This is easy to test, I believe I also provided JCL in the sample that is i github. I don't consider this a show stopper but would be nice if we knew it actually copied well enough to retain its exactable bits. 

Here are two additional ways to JCL to run the program:
```
  tasks:
    - name: Execute with mvscmd
      ansible.builtin.shell: mvscmd --pgm=HELLOPGM  --steplib=ANSIBLE.COBOL.LOADLIB --sysprint=* --stderr=* --stdout=*
      register: result

    - name: Result
      debug:
         msg: "{{result}}"

    - name: Execute program
      zos_mvs_raw:
        program_name: HELLOPGM
        auth: false
        verbose: true
        dds:
          - dd_data_set:
              dd_name: steplib
              data_set_name: ANSIBLE.COBOL.LOADLIB
              disposition: shr
          - dd_output:
              dd_name: sysprint
              return_content:
                type: text
          - dd_output:
              dd_name: stderr
              return_content:
                type: text
          - dd_output:
              dd_name: stdout
              return_content:
                type: text
      register: result

    - name: Result
      debug:
         msg: "{{result}}"
```

Reply:
Thanks, added the check. Only checks that rc == 0, if it cannot execute it rc != 0.